src/com/amazon/corretto/crypto/provider/AmazonCorrettoCryptoProvider.java

```diff
@@ -121,6 +121,9 @@ private void buildServiceMap() {
               singletonMap("ThreadSafe", "true"),
               "DEFAULT")
           .setSelfTest(LibCryptoRng.SPI.SELF_TEST);
+      // Following line is a workaround to ensure that the SecureRandom service
+      // is seens as ThreadSafe by SecureRandom, when using the alias name DEFAULT
+      setProperty("SecureRandom.DEFAULT ThreadSafe", "true");
```

Comment:
Could you add a test that verifies both "DEFAUTL" and "LibCryptoRng" have "ThreadSafe" property set to true?